extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -175,18 +241,11 @@ export class AmpSelector extends AMP.BaseElement {
   }
 
   /**
-   * Handles the change event for the selectables.
-   * @param {!Event} event
+   * Handles user selection on an option.
+   * @param {!Element} el The element selected.
    */
-  clickHandler_(event) {
-    let el = dev().assertElement(event.target);
-    if (!el) {
-      return;
-    }
-    if (!el.hasAttribute('option')) {
-      el = closest(el, e => e.hasAttribute('option'), this.element);
-    }
-    if (!el || el.hasAttribute('disabled')) {
+  onOptionSelected_(el) {
```

Comment:
this is an action that happens after a click as opposed to an action that happens after selection - this probably determines if a selection should happen, hence the name is misleading